### ISSUE TITLE
Get `condition` in content template?

### ISSUE THREAD

**Issue by shakiba** [juliancwirko/react-s-alert#50]
Is there anyway I can get `condition` in content template?  I can extract it from class names passed in props, but if there is anyway to do it without hacking it would be great.

**Comment by yongzhoulu** [comment]
need this feature

**Comment by yongzhoulu** [comment]
I made a PR. Feel free to edit it.
https://github.com/juliancwirko/react-s-alert/pull/51

**Comment by shakiba** [comment]
@yongzhoulu Great job!

**Comment by juliancwirko** [comment]
Hi thanks, can you explain the use case for that?

**Comment by yongzhoulu** [comment]
In my case. I want different color and different icon inside alert container.
![default](https://user-images.githubusercontent.com/30485129/31605793-c9cca264-b299-11e7-909c-fcaded1a9a2c.PNG)
![nnn](https://user-images.githubusercontent.com/30485129/31605795-cb04e7ae-b299-11e7-8961-feb1355195b4.PNG)

In this case, i use customTemplate. And i need to know the condition to choose which icon and color i use.
You provide the condition api in alertContent in `classNames`, of couse i can extract from it.
It would be easier for me when having `condition` api. Or i must use customFields to provide this information.

**Comment by yongzhoulu** [comment]
@juliancwirko hope this scenario is well explained to you

**Comment by juliancwirko** [comment]
Most of the cases are fulfilled by styling and using provided class names, but yes, I understand that it could be useful when you want to change alert's structure depending on condition.

Ok, I'll work on the lib today and I'll see how we can merge that. Thanks.

**Comment by yongzhoulu** [comment]
That would be great. Thanks for your time.

**Comment by 3CordGuy** [comment]
Seems like just exposing the `condition` prop would suffice for most of these cases. What's the status on this? I would just like to have a custom icon as well depending on the called condition method. Seems overkill to have to specify that in a custom field object.  :)

**Comment by juliancwirko** [comment]
Sorry, not ready yet :/ I'll try in the next week.

**Comment by 3CordGuy** [comment]
🤔  #51 is all we need. I guess I'll extract it from the className string until then.

**Comment by juliancwirko** [comment]
yes, it seems to be the solution here, but I need to spend some time on verification, testing etc.
